app/helpers/dynamodb.py

```diff
@@ -13,14 +19,14 @@ def save_item(self, kml_admin_id, file_id, timestamp):
         self.table.put_item(
             Item={
                 'adminId': kml_admin_id,
-                'fileId': file_id,
+                'file_id': file_id,
```

Comment:
Naming convention here, `adminId` and `file_id` should use the same convention, at best snake case. However I don't know if this has any external impact changing these names ?